### DISCUSSION TITLE
Search by Tag or Category

### DISCUSSION THREAD

**Discussion by tomoem** [OrchardCMS/OrchardCore#12138] [Q&A] [answered]
Hi,
I'm trying to make the search work with tags and categories, in Orchard Core Full CMS (Blog recipe).

I've edited the Lucene Index to include the content types: "Tag", "Category" and "Taxonomy".
In the content type "Blog Post", edited the fields "Tags" and "Category" and selected: "Include this element in the index", "Stored" and "Analyzed".
In the Search Setting, Default searched fields, add the following: `Content.ContentItem.FullText, BlogPost.Tags.FullText, BlogPost.Category`

But with no success so far. How can I make the search work with tags and categories?
Thanks

**Comment by jsalgado10** [comment, marked as answer]
I had the same scenario and was able to add the fields under the Use custom full text option.

![image](https://user-images.githubusercontent.com/16822143/185465902-af0df57a-1a5e-4a7a-97df-a7e128c180ef.png)

**Reply by jsalgado10** [reply]
this will need to be done on the Blog Post content type

**Reply by tomoem** [reply]
Thanks!
